SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealMetadata.h

```diff
@@ -59,7 +59,7 @@ struct UnrealMetadata : Component
 
 	FORCEINLINE UClass* GetNativeEntityClass()
 	{
-		if (UClass* Class = FindObject<UClass>(ANY_PACKAGE, *ClassPath))
+		if (UClass* Class = LoadObject<UClass>(nullptr, *ClassPath))
```

Comment:
@m-samiec loading this from the transient package should be fine right?

Reply:
There's a chance this could cause issues with sublevels if we hear about a known sublevel entity type (the level script) before the client has a chance to load the level itself. This should get fixed with proper sublevel/qbi support though.